website_form_require_legal/static/src/js/options.js

```diff
@@ -35,7 +35,6 @@ odoo.define("website_form_require_legal.form_editor", function (require) {
                 $(template).html(
                     qweb.render("website_form_require_legal.s_website_form_legal", {
                         labelWidth: labelWidth,
-                        termsURL: "terms",
```

Comment:
Why removing this default value?

Reply:
I confirm that this is working with a sane default `/terms`, so please restore it with that value.

Reply:
done!